src/routes/docs/products/databases/permissions/+page.markdoc

```diff
@@ -23,6 +23,10 @@ Configure collection level permissions by navigating to **Your collection** > **
 Document level permissions grant access to individual documents.
 If a user has read, create, update, or delete permissions at the document level, the user can access the **individual document**.
 
+{% info title="Does Appwrite support Row Level Security (RLS)?" %}
+Document level security in Appwrite is similar to Row Level Security (RLS) found in database systems like PostgreSQL, as both provide granular access control to individual records. However, they work differently: RLS uses policies that automatically filter results based on conditions, while Appwrite requires explicitly setting permissions on each document.
```

Comment:
> _explicitly_ setting permissions

Does this sound negative to you? Can you reword it to not sound like a drawback/flaw?